### ISSUE TITLE
Alot of Config settings are client side

### ISSUE THREAD

**Issue by itsmagu** [alRex-U/ParCool#201]
**Summary**
Alot of Config settings are client side

**Running Environment**
+ Minecraft Version: 1.19.2
+ ParCool Version: 3.1.0.3-R
+ Forge Version (optional): 43.3.0

**Detail**
Hi Love the mod so far, I have it installed on our school server!
We noticed that the config file for the server lacks alot of settings compared to the client.
We also use the feather mod, but this can be disabled by clients! We would like to be able
to lock this via the server since we use it to restrict players with alot of armor...
Otherwise, excellent mod. It feels give minecraft some new gameplay that we all enjoy!

**Additional Data**

Server config:
![image](https://github.com/alRex-U/ParCool/assets/95685391/7b7e0f8f-7605-4e81-ada3-40342f8ff4ad)

Client Config
![image](https://github.com/alRex-U/ParCool/assets/95685391/ea63e65c-39a5-4058-a13f-d7690fac346b)
![image](https://github.com/alRex-U/ParCool/assets/95685391/837978d1-8d5b-44af-8474-a56afcbd4128)

**Comment by itsmagu** [comment]
Yes that would suffice

**Comment by itsmagu** [comment]
Hi again, Just wanted to add that alot of server config options is not followed.
Here I disable Climb Poles
![image](https://github.com/alRex-U/ParCool/assets/95685391/dab2be4e-5346-4e82-8da8-b2660e6a9f21)
And Enable Limitations
![image](https://github.com/alRex-U/ParCool/assets/95685391/6b4bf048-2dbe-43ac-b8c3-3ea46887632c)
![image}(https://cdn.discordapp.com/attachments/1087707373436674098/1157686197754265640/Serverconfig_is_not_followed.gif?ex=651982ca&is=6518314a&hm=6bf3390fa988aca3ceea425317524dfde3f2084dec4b4acd79508204bfbf5684&)

**Comment by itsmagu** [comment]
In fact none of these are in effect : /
![image](https://github.com/alRex-U/ParCool/assets/95685391/81549b2b-fe57-4630-8bc7-0b8ea3f56e12)

**Comment by alRex-U** [comment]
You rebooted the game?

**Comment by itsmagu** [comment]
yes, have been running these settings for days, currently I have not told anyone how to bypass it in settings so its fine for now

**Comment by alRex-U** [comment]
If so this may be a bug. I will check it later. Is this in 1.19.2 game right?

**Comment by itsmagu** [comment]
correct, newest version

**Comment by alRex-U** [comment]
In next update, limitation is deleted and server config will provide this feature
